CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
-## v2.6.3 - 2024-11-TODO
+
+## v2.6.4 - 2025-05-22
+
+### Fixes
+- Dependencies: Pin click to `~=8.1.0` (#6888) [[e1d55fa]](https://github.com/aiidateam/aiida-core/commit/e1d55fa0d6c268db7a25e9740679a5a0fbe412b8)
+- Fixing doc references after `click` 8.2 release (#6880) [[5168032]](https://github.com/aiidateam/aiida-core/commit/5168032b810aef94ca91c91859e53e1752c72273)
```

Comment:
```suggestion
```

Let's remove this, it's irrelevant to the readers of the changelog and does not touch code.